docs/irc_operators.md

```diff
@@ -0,0 +1,78 @@
+# IRC Operators
+
+This chapter describes useful information about the bridge for IRC operators or IRC users.
+
+The IRC bridge provides each Matrix user with one IRC connection in order to bridge them "natively"
+into the IRC network, as so they can largely be treated as real users. Due to the 1:1 connection system,
+it is often useful that the IRCD network provide the bridge host with a more relaxed ILINE limit depending
```

Comment:
IRCD and ILINE are terms that would benefit from a short explanation.
One needs to be very deep in the IRC terminology to be familiar with those.

Reply:
I think this is fine because the document mentions that you are expected to be an IRC operators. I will see if I can link to another page explaining it though.